src/utils/dateUtils.js

```diff
@@ -0,0 +1,17 @@
+const weekDayNamesEng = [
+  'Sunday',
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday',
+];
+
+const makeGetDayOfTheWeek = (weekDayNames) => (date) => weekDayNames[date.getDay()];
```

Comment:
Nice!

But as **weekDayNamesEng** is a constant you could probably skip additional closure and read value directly like: 
`const getDayOfTheWeek = date => weekDayNamesEng[date.getDay()];`

What do you think ? Am I missing something ? Probably you wanted to reuse **makeGetDayOfTheWeek** somewhere else ?